test/integration/virtual-rules/nested-interactive.js

```diff
@@ -109,7 +109,7 @@ describe('nested-interactive virtual-rule', function() {
 
     var results = axe.runVirtualRule('nested-interactive', node);
 
-    assert.lengthOf(results.passes, 0);
+    assert.lengthOf(results.passes, 1);
```

Comment:
I have no explanation for why this test wasn't erroring before... There are two applicable nodes in this tree, one passes, the other fails.

Reply:
this blew my mind but I gave up trying to understand why

Reply:
Yeah, likewise.